Mod2/Labs/Week2/DataRelationships.md

```diff
@@ -0,0 +1,39 @@
+## Lab Objectives
+* Add to your library database, create a many-to-many relationship between Books and Authors
+
+We are going to continue building on our Library database by adding a many-to-many relationship between books and authors. This relationship should be many-to-many because a book can have many authors and an author can write many books.
```

Comment:
Is this true for "normal" books? Textbooks maybe. I think you mentioned you're a reader, but when I think of books, I think of a sole author.

Reply:
Good call-out. You're right, don't think it's super common, but there are at least some books that have two authors and anthologies that have many many. I'll put an example in the lesson :)